napari/__init__.py

```diff
@@ -1,16 +1,28 @@
-import warnings
+import os
+from pathlib import Path
+from qtpy import API_NAME
 
-vispy_warning = "VisPy is not yet compatible with matplotlib 2.2+"
+if API_NAME == 'PySide2':
+    # Set plugin path appropriately if using PySide2
+    import PySide2
 
-with warnings.catch_warnings():
-    warnings.filterwarnings(
-        "ignore", category=UserWarning, message=vispy_warning
+    os.environ['QT_PLUGIN_PATH'] = str(
+        Path(PySide2.__file__).parent / 'Qt' / 'plugins'
     )
-    from .viewer import Viewer
+elif API_NAME == 'PyQt5':
+    # Set plugin path appropriately if using PyQt5
+    import PyQt5
 
-from ._view import view
+    os.environ['QT_PLUGIN_PATH'] = str(
+        Path(PyQt5.__file__).parent / 'Qt' / 'plugins'
+    )
+else:
+    raise (ValueError('qtpy backend must either be PyQt5 or PySide2'))
```

Comment:
i dont think we should enforce which backend people are using; at most give a warning

we should, however, enforce a minimum version requirement so this should be done here

Reply:
I don't understand `enforce a minimum version requirement` but not `which backend`? Should I just remove this error entirely?

Reply:
the error itself should be removed

we _should not_ error if a backend other than PyQt5 or PySide2 is used

we _should_ error if the Qt version does not meet our minimum requirements

Reply:
sounds good, I have removed this error